doc/stdlib/pkg.rst

```diff
@@ -10,8 +10,9 @@ to use them, you'll need to prefix each function call with an explicit ``Pkg.``,
 .. function:: dir() -> String
 
    Returns the absolute path of the package directory.
-   This defaults to ``joinpath(homedir(),".julia")`` on all platforms (i.e. ``~/.julia`` in UNIX shell syntax).
-   If the ``JULIA_PKGDIR`` environment variable is set, that path is used instead.
+   This defaults to ``joinpath(homedir(),".julia","$(VERSION.major).$(VERSION.minor)")`` on all platforms
+   (i.e. ``~/.julia/0.4`` in UNIX shell syntax).  If the ``JULIA_PKGDIR`` environment variable is set, then
```

Comment:
This should be `v0.4`, not `0.4`.  (And since it's for the `release-0.3` branch, it should actually be `v0.3`.)  Same on the line below; there is a `v` missing.